src/bun.js/api/server.zig

```diff
@@ -1035,16 +1037,16 @@ fn NewRequestContext(comptime ssl_enabled: bool, comptime debug_mode: bool, comp
                 var request_js = this.request_js_object.?.value();
                 request_js.ensureStillAlive();
 
-                this.request_js_object = null;
+                // this.request_js_object = null;
                 defer request_js.ensureStillAlive();
                 defer JSC.C.JSValueUnprotect(this.server.globalThis, request_js.asObjectRef());
                 // User called .blob(), .json(), text(), or .arrayBuffer() on the Request object
                 // but we received nothing or the connection was aborted
                 if (request_js.as(Request)) |req| {
                     // the promise is pending
-                    if (req.body == .Locked and req.body.Locked.action != .none and req.body.Locked.promise != null) {
```

Comment:
Was this a bug? This feels unrelated

Reply:
``` javascript
export default {
  async fetch(request) {
    console.log(".json()", request.json());
    return new Response("Ok");
  }
}
```

In this case, it would output TypeError: Request aborted, if not commenting it out.